### PR TITLE
Create new manage command: list_repeater_domain_admins

### PR DESCRIPTION
**Pull request by joxl**
Add new manage command `list_repeater_domain_admins` to acquire a list of domains (and their admin emails) which use repeaters.  This is in preparation for determining at-risk domains for the CA trust chain update in the upcoming `certifi` library upgrade (#29821).

Jira: [SAAS-12157](https://dimagi-dev.atlassian.net/browse/SAAS-12157)

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### QA Plan

No QA.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/motech/repeaters/management/commands/list_repeater_domain_admins.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    help = """Shows names and admin emails of Domains with Repeaters."""
+
+    def add_arguments(self, parser):
+        parser.add_argument("--csv", action="store_true", default=False,
+            help="Write output as CSV data instead of padded table.")
+        parser.add_argument("-o", "--output", metavar="FILE", type=FileType("w"),
+            default=self.stdout, help="Write output to %(metavar)s (default=STDOUT).")
+
+    def handle(self, *args, **options):
+        log.debug("fetching repeaters")
```
Comment (biyeun) [inline corehq/motech/repeaters/management/commands/list_repeater_domain_admins.py:22]: for management commands I feel it's best to use `self.stdout.write`. Are you purposely wanting these outputs to go to logs/athena?

**Review by kaapstorm — APPROVED** [review]
What @biyeun said. Adding an argument for verbosity and printing to stdout might work better than debug logging. Otherwise looks good to me.

File: corehq/motech/repeaters/management/commands/list_repeater_domain_admins.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                emails.add(admin.email if admin.email else admin.username)
+            table.add_row([domain, ", ".join(sorted(emails))])
+        table.sort()
+        if options["csv"]:
+            table.write_csv(options["output"])
+        else:
+            options["output"].write(table.render())
+
+
+class Table:
```
Comment (kaapstorm) [inline corehq/motech/repeaters/management/commands/list_repeater_domain_admins.py:39]: Probably unreasonable to add a new requirement just for one management command, but I love [tablib](https://pypi.org/project/tablib/).

```python
>>> data = tablib.Dataset(headers=['First Name', 'Last Name', 'Age'])
>>> for i in [('Kenneth', 'Reitz', 22), ('Bessie', 'Monke', 21)]:
...     data.append(i)

>>> data.export('df')
  First Name Last Name  Age
0    Kenneth     Reitz   22
1     Bessie     Monke   21

>>> print(data.export('rst'))                                                            
+---------+-------+-----+
|  First  |  Last | Age |
|   Name  |  Name |     |
+=========+=======+=====+
| Kenneth | Reitz | 22  |
+---------+-------+-----+
| Bessie  | Monke | 21  |
+---------+-------+-----+

>>> print(data.export('csv'))                                                                                                   
First Name,Last Name,Age
Kenneth,Reitz,22
Bessie,Monke,21

```
(I think I forgot to click "Start a review" and I lost this comment, so trying again. Sorry if you got notified of the same comment twice.)

**Comment by joxl** [comment]
Thanks for the feedback @biyeun  and @kaapstorm. I'm dropping this effort for now to work on ES, so I'm going to close this PR.  I'll leave the branch out there in case someone else (or myself) picks it back up at a later time.

**Pull request closed without merging**